js/data/locale/root/langname.jf

```diff
@@ -1,4 +0,0 @@
-{
-    "generated": true,
-    "language.name": "Root"
-}
```

Comment:
I don't think this one should be removed. If the locale has no language tag or the language tag does not have a langname.jf, then this one takes over....

Reply:
then, it should be located under locale/ directory, not locale/root/ directory. no?

Reply:
Ah yes, I missed that part.